stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -571,6 +576,7 @@
                           (pox-addr (tuple (version (buff 1)) (hashbytes (buff 32))))
                           (start-burn-ht uint)
                           (lock-period uint)
+                          (signer-sig (buff 65))
```

Comment:
Synthetic pox events are generated using argument indexes rather than named values, so it's generally easier to add new args to the end of the list. Not a deal breaker in this PR because `signer-sig` isn't yet added to the pox events, but I'm working on adding that and our PRs will conflict due to the ordering.

Reply:
Conflict PR: https://github.com/stacks-network/stacks-core/pull/4329

I can modify my PR and change the event arg indexes, no problem. Just an FYI

Reply:
No problem, let's merge your PR and I'll update mine.

Reply:
I have no real preference on arg order. Is your PR the only real place we need to worry about arg index? I'd probably just update that synthetic events code to change the signer_key index, but if there are other downstream issues then it's probably better for me to change the argument order on pox-4.

Reply:
Thanks!